### PR TITLE
added button component style & some minor typos

### PR DESCRIPTION
**Pull request by aishwarya-bm**
added button component style & some minor typos in color variables.

check out the preview video here:
https://user-images.githubusercontent.com/66695068/151754867-e1a842d4-88ed-4bcd-b3c3-24b00fc2b507.mp4

### CODE REVIEW

File: stylesheets/components/button.css [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+.btn-link:hover {
+  background-color: var(--primary-lighter);
+}
+
+.btn-success {
+  background-color: var(--success-txt);
+}
+
+.btn-success:hover {
+  background-color: var(--success-bg);
```
Comment (sadathshariff) [inline stylesheets/components/button.css:52]: You can give the text some color when you hover over it the text is not clearly visible

**Review by sadathshariff — COMMENTED** [review]
Everything else looks fine

File: stylesheets/components/button.css (hunk 1 above)
Comment (tanishq20) [reply stylesheets/components/button.css:52]: Adding on this u can also give some darkish green background with the text color while hovering.

**Review by tanishq20 — COMMENTED** [review]
One minor change only

File: stylesheets/components/button.css (hunk 1 above)
Comment (aishwarya-bm) [reply stylesheets/components/button.css:52]: Thanks guys, I have made the changes! :)

**Pull request merged**